corehq/apps/accounting/templates/accounting/partials/new_stripe_card_template.html

```diff
@@ -0,0 +1,100 @@
+{% load hq_shared_tags %}
+{% load i18n %}
+
+<script type="text/html" id="new-stripe-card-template">
+    <div class="modal hide" id="card-modal">
+        <div class="modal-header">
+            <a class="close" data-dismiss="modal">&times;</a>
+            <h3>{% trans "Add New Credit Card" %}</h3>
+        </div>
+        <div class="modal-body">
+            <input type="hidden"
+                   name="stripeToken"
+                   data-bind="value: token" />
+            <div class="alert alert-success"
+                 data-bind="visible: isProcessing">
+                {% trans "Processing your request" %}
+            </div>
+            <div class="alert alert-error"
+                 data-bind="visible: errorMsg() != '',
+                            text: errorMsg">
+            </div>
+            <div class="control-group">
+                <label class="control-label">
+                    {% trans 'Card Number' %}
+                </label>
+                <div class="controls">
+                    <input type="text"
+                           data-bind="value: number, disable: isProcessing"
+                           size="20"
+                           placeholder="xxxx-xxxx-xxxx-xxxx", />
+                </div>
+            </div>
+
+            <div class="control-group">
+                <label class="control-label">
+                    {% trans 'CVC' %}
```

Comment:
isn't there the ability to leave a hint for the translator? this would be a good place for that.